sphinxcontrib/chapeldomain.py

```diff
@@ -508,14 +508,14 @@ class ChapelModuleLevel(ChapelObject):
     @property
     def chpl_type_name(self):
         """Returns iterator or procedure or '' depending on object type."""
-        if not self.objtype.endswith('function'):
+        if self.objtype == 'type':
+            return 'type'
+        elif not self.objtype.endswith('function'):
```

Comment:
You'll want to update the comment, too.

Out of curiosity, was it organizationally better to have the type case first, or was it causing problems?